content/blog/2024/dei-leadership-strategy/index.md

```diff
@@ -43,9 +43,9 @@ Here are some steps we aim to take to improve the diversity of our leadership ac
-3. **Make a plan for improving leadership diversity within our team**. Whatever mechanisms we create for diverse leadership, we know that one of them needs to be improving our diversity at a staff leadership level. Our staff are the ones that spend the most time working on - and have the most leverage over - our strategy and mission, and it's important that our staff leadership is a good representation of the diverse perspectives of the communities we serve. This might mean investing more heavily in reaching broad applicant pools for new positions or seeking external consultation for how we can avoid unintentionally sending exclusionary signals to others in our communications and outreach.
-4. **Continue improving our systems of delivery and work**. Finally, we believe that a crucial aspect of improving the diversity in our organization is to improve our systems for delivery, reliability, and accountability across the organization. We have [made a lot of progress in improving our delivery systems](../delivery-improvements/index.md) and will continue this improvement as an organization.
+3. **Make a plan for improving leadership diversity within our team**. Whatever mechanisms we create for diverse leadership, we know that one of them needs to be improving our diversity at a staff leadership level. Our staff are the ones that spend the most time working on - and have the most leverage over - our strategy and mission, and it's important that our staff leadership is a good representation of the diverse perspectives of the communities we serve. This might mean investing more heavily in reaching broad applicant pools for new positions, seeking external consultation for how we can avoid unintentionally sending exclusionary signals to others in our communications and outreach and developing incentives to retain our talent over time.
+4. **Continue improving our systems of delivery and work**. Finally, we believe that a crucial aspect of improving the diversity in our organization is to continue transforming our systems for delivery, reliability, and accountability across the organization. This effort is a crucial step for fostering an inclusive and equitable culture. We have [made a lot of progress in improving our delivery systems](../delivery-improvements/index.md) and will continue this improvement as an organization.
 
 We hope that this post both makes transparent a distinct lack of diversity within our organizational leadership and provides clarity for some of our plans to improve.
-We are excited about the organization that we have built, as well as its culture.
+We are excited about the organization that we have built so far, as well as being fiercely proud of our open-source culture and development practises.
```

Comment:
> fiercely proud

:heart_eyes:

Reply:
I'm pretty sure our whole team feels this way ❤️‍🔥